app/controllers/api/connect/v3/systems/systems_controller.rb

```diff
@@ -3,6 +3,17 @@ class Api::Connect::V3::Systems::SystemsController < Api::Connect::BaseControlle
   before_action :authenticate_system
 
   def update
+    if params[:online_at].present?
+      params[:online_at].each do |online_at|
+        dthours = online_at.split(':')
+        @system_uptime = SystemUptime.create!(system_id: @system.id, online_at_day: dthours[0], online_at_hours: dthours[1])
```

Comment:
conversion from `string` to `binary(24)` works like this? I'm just reading the code here and was wondering if `1111:string` is implicitly converted to `1111:binary`  and not the ASCII representation `00110001001100010011000100110001:binary` or similar in UTF8. Did you check?

Reply:
I tested and if paylod is something like 
, [2024-01-22T21:11:08.302704 #5482]  INFO -- :   Parameters: {"hostname"=>"sle15sp5", "distro_target"=>"sle-15-x86_64", "hwinfo"=>{"hostname"=>"sle15sp5", "cpus"=>2, "sockets"=>2, "hypervisor"=>"KVM", "arch"=>"x86_64", "uuid"=>"9abd4d7c-3543-46c2-bcb9-784200c0e8aa", "cloud_provider"=>"", "mem_total"=>1964}, "online_at"=>["2023-12-25:000011111111111111111111", "2023-12-31:000011111111111111111111", "2024-01-16:000011111111111111111111", "2024-01-18:000011111111111111111111"], "system"=>{"hostname"=>"sle15sp5"}}

The data entered in the table is 
Database changed
MariaDB [rmt_development]> SELECT * FROM system_uptimes;
+-----------+---------------+--------------------------+----------------------------+----------------------------+
| system_id | online_at_day | online_at_hours          | created_at                 | updated_at                 |
+-----------+---------------+--------------------------+----------------------------+----------------------------+
|         1 | 2023-12-25    | 000011111111111111111111 | 2024-01-22 21:11:08.405177 | 2024-01-22 21:11:08.405177 |
|         1 | 2023-12-31    | 000011111111111111111111 | 2024-01-22 21:11:08.415410 | 2024-01-22 21:11:08.415410 |
|         1 | 2024-01-16    | 000011111111111111111111 | 2024-01-22 21:11:08.425858 | 2024-01-22 21:11:08.425858 |
|         1 | 2024-01-18    | 000011111111111111111111 | 2024-01-18 06:40:07.415815 | 2024-01-18 06:40:07.415815 |
+-----------+---------------+--------------------------+----------------------------+----------------------------+
4 rows in set (0.001 sec)

Reply:
awesome! Thanks for checking